src/js/components/explorer/detail/visualization/ExplorerVisualization.jsx

```diff
@@ -129,7 +132,8 @@ export default class ExplorerVisualization extends React.Component {
                 {visualization}
 
                 <div className="explorer-vis__disclaimer">
-                    All dollar amounts shown here represent agency reported obligated amounts
+                    <p>All dollar amounts shown here represent agency reported obligated amounts</p>
+                    <p><span className="explorer-vis_bold">Unreported Data*:</span> Unreported amounts are calculated using the difference in the total obligated amount from the <a href="https://max.omb.gov/maxportal/document/SF133/Budget/FACTS%20II%20-%20SF%20133%20Report%20on%20Budget%20Execution%20and%20Budgetary%20Resources.html">Report on Budget Execution and Budgetary Resources</a> and the total obligated amount reported by agencies to USAspending.gov.</p>
```

Comment:
Since this link goes to a non-Fiscal Service site, it should trigger the redirect modal. Example here for Database Snapshots in the Download Center navigation dropdown: https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/components/sharedComponents/header/DropdownItem.jsx#L76